pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/permissions_role_create_cluster/index.md

```diff
@@ -0,0 +1,240 @@
+      - Ref: DKPBootstrapRole
+    Type: AWS::IAM::ManagedPolicy
+  DKPBootstrapRole:
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+        - Action:
+          - sts:AssumeRole
+          Effect: Allow
+          Principal:
+            Service:
+            - ec2.amazonaws.com
+        - Action:
+            - sts:AssumeRole
+          Effect: Allow
+          Principal:
+                AWS: arn:aws:iam::MYAWSACCOUNT:root
+        Version: 2012-10-17
+      RoleName: dkp-bootstrapper-role
+    Type: AWS::IAM::Role
+ ```
+
+
+To create the resources in the cloudformation stack, copy the contents above into a file and run the following command after replacing `MYFILENAME.yaml` and `MYSTACKNAME` with the intended values:
+
+```bash
+aws cloudformation create-stack --template-body=file://MYFILENAME.yaml --stack-name=MYSTACKNAME --capabilities CAPABILITY_NAMED_IAM
+```
+
+## Leverage the Role
+### Temporary User Access Keys via STS 
+The created `dkp-bootstrapper-role` can be assumed by IAM users for temporary credentials via STS by running the command below:
+
+<p class="message--note"><strong>NOTE: </strong>You must replace MYAWSACCOUNT with an AWS Account ID number ie: 111122223333 </p>
+
+aws sts assume-role --role-arn arn:aws:iam::MYAWSACCOUNT:role/dkp-bootstrapper-role --role-session-name EXAMPLE 
+Which returns something similar to this:
+
+
+{
+    "Credentials": {
+        "AccessKeyId": "ASIA6RTF53ZH5B52EVM5",
+        "SecretAccessKey": "BSssyvSsdfJY74jubsadfdsafdsaH7x1L+8Vk/",
+        "SessionToken": "IQoJb3JpZ2z5cyChb9PtJvP0S6KAi",
+        "Expiration": "2022-07-14T20:19:13+00:00"
+    },
+    "AssumedRoleUser": {
+        "AssumedRoleId": "ASIA6RTF53ZH5B52EVM5:test",
+        "Arn": "arn:aws:sts::MYAWSACCOUNTID:assumed-role/dkp-bootstrapper-role/test"
+    }
+}
+And then export the following environment variables with the results:
+
+```bash
+export AWS_ACCESS_KEY_ID=(.Credentials.AccessKeyId)
+
+export AWS_SECRET_ACCESS_KEY=(.Credentials.SecretAccessKey)
+
+export AWS_SESSION_TOKEN=(.Credentials.SessionToken)
+ ```
+
+These credentials are short lived and would need to be updated in the bootstrap cluster
+
+### Use EC2 Instance Profiles
+The created `dkp-bootstrapper-role` can be assumed by an ec2 instance a user would run `dkp create cluster` commands from. To do this, specify the IAM Instance Profile `DKPBootstrapInstanceProfile` on creation. 
+
+### Use Access Keys
+AWS administrators can attach the dkp-bootstrapper-policy to an existing IAM user and authenticate with Access Keys on the work station they would run dkp create cluster commands from by exporting the following environment variables with the appropriate values for the IAM user.  
+
+
+export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+export AWS_DEFAULT_REGION=us-west-2
+
+### Best practices 
+A system administrator should always consider best practices for Access Keys.
```

Comment:
This is missing a link https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html